### ISSUE TITLE
Selection marker needs a small change....! 

### ISSUE THREAD

**Issue by douxlagithub** [handsontable/handsontable#1131]
HT selection marker needs to be evaluated back. It feels comfortable when we select data in excel because the marker is strongly pointing us to the selected data.  This happens because of the border size and solid black color used by the marker.  Then during the selection, the current rectangle is disappearing and replaced by selection marker.  

This picture is captured from my modified HT (Class 'current', border '2', color '#000000') then (Area border '2', color '#000000') and the last (Corner border '1' [handle bar]).

Let's look at the following (Current working):
![selection_fix](https://f.cloud.github.com/assets/4103210/1442278/891f8488-41ba-11e3-9962-3f79a67e404b.png)

In short, I wish the selection marker does exactly like excel do.  This issue will definetely enhance user Experience.  I hope beta 4 will include this idea and next releases.

**Comment by AMBudnik** [comment]
We are closing issues related to feature requests to create a full document based on that ideas from all our sources. It is avaiable is our [wiki section](https://github.com/handsontable/handsontable/wiki/Feature-requests).

**Comment by AMBudnik** [comment]
As there aren't many comments on closed issues, we decided to reopen the feature requests as Github Discussions. 

Please feel free to share any feedback that will allow us to compose the full list of requirements for the proposed change.
